src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -356,8 +365,8 @@ static Task EmptyErrorCallback(Exception exception)
         ConcurrentDictionary<Task, Task> pipelineInvocationTasks;
         string fullPath;
         EntityInfoInternal entity;
+        long refCount;
```

Comment:
any reason for choosing a long over int?

Reply:
Nothing other than I was usually using the Int64 based overload. Happy to change though